Gems/Atom/Feature/Common/Assets/Materials/Types/MaterialFunctions/EvaluateEnhancedSurface.azsli

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Atom/Features/PBR/Surfaces/EnhancedSurface.azsli>
+#include <Atom/Features/MatrixUtility.azsli>
+
+void EvaluateEnhancedSurface(
```

Comment:
Unclear why this needs to be factored out from EnhancedSurface_ForwardPass.azsli since it's only used once

Reply:
Because in the future, the material pipeline for how the data is used may change (e.g. VR, low-end pipelines, etc.). The decoupling here is intended to make this future path possible